tests/drivers/can/api/board/mimx93_evk_a55.overlay

```diff
@@ -0,0 +1,28 @@
+/{
+    chosen{
+        zephyr,canbus = &flexcan2;
+    };
+};
+
+&lpi2c2 {
+    status = "okay";
+};
+
+&mfd0 {
+    status = "okay";
+};
+
+&gpiohog_exp_sel {
+    status = "okay";
+    output-low;
+};
+
+&can_phy0 {
+    status = "okay";
+};
+
+&flexcan2 {
+	bus-speed = <1000000>;
+    bus-speed-data = <5000000>;
```

Comment:
```suggestion
```

These properties are not needed anymore. The tree wide default is set via Kconfig.

Reply:
removed